problemsets-mat223/modules/module11-exercises.tex

```diff
@@ -1,17 +1,46 @@
 \begin{exercises}
+    %Patrick:
+        %30min
+        %60min
+        %30min
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob Give the following concepts a definition with the book closed.
+	    \begin{enumerate}
+	        \item The range (or image) of a linear transformation $T: V \to W$.
+	        \item The rank of of a linear transformation $T: V \to W$.
+	        \item The null space( or kernel) of a linear transformation $T: V \to W$.
+	        \item The nullity of  of a linear transformation $T: V \to W$.
+	        \item The row space, the column space and the null space of a matrix $M$.
+	        
+	    \end{enumerate}
+		\prob  From the definitions, find the range and null space of the following linear transformations.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+		    \item   $\mathcal P:\R^2\to\R^2$, a linear transformation projecting onto the line $y=ax+b$, for some $a,b \in R$.
+			\item   $\mathcal R:\R^2\to\R^2$, a linear transformation rotating clockwise by some degree $\theta\in\R$.
+			\item   $\mathcal F:\R^2\to\R^2$, a linear transformation flipping over the line $y=ax+b$, for some $a,b \in R$.
+			\item   $\mathcal S:\R^2\to\R^2$, a linear transformation stretching by $k$, for some non-zero real number $k$.
+		\end{enumerate}
+		
+		\prob Find the null space of $M=\mat{1&2&1\\3&1&-2\\8&6&-2}$.
+		
+		\prob Let $\mathcal T$ be the transformation induced by the matrix $M=\mat{7&5\\-2&-2}$,
+	    and let $\vec v=3\xhat-3\yhat$. Compute $\mathcal T_{M}\vec v]_{\mathcal E'}$ and $\mathcal T(\vec v)$.
```

Comment:
Your brackets don't match. (Are you proofreading the problems before you commit?)